client/components/foldable-card/docs/example.jsx

```diff
@@ -61,7 +71,9 @@ export default class FoldableCardExample extends PureComponent {
 								</div>
 							</div>
 						}
+						// eslint-disable-next-line wpcalypso/jsx-classname-namespace
```

Comment:
I wonder if we could fix these in another way, for example by passing a `<Button />` component instead of `<button />` with a `classname`.

Reply:
Possibly, but given it's devdocs I didn't want to spend too much time on this. Dinner was calling ;)